internal/cmd/api-key/command_delete.go

```diff
@@ -28,18 +30,67 @@ func (c *command) newDeleteCommand() *cobra.Command {
 
 func (c *command) delete(cmd *cobra.Command, args []string) error {
 	c.setKeyStoreIfNil()
-	apiKey := args[0]
 
-	promptMsg := fmt.Sprintf(errors.DeleteResourceConfirmYesNoMsg, resource.ApiKey, apiKey)
-	if ok, err := form.ConfirmDeletion(cmd, promptMsg, ""); err != nil || !ok {
+	if err := c.checkExistence(cmd, args); err != nil {
 		return err
 	}
-	httpResp, err := c.V2Client.DeleteApiKey(apiKey)
 
+	if ok, err := form.ConfirmDeletionYesNo(cmd, resource.ApiKey, args); err != nil || !ok {
+		return err
+	}
+
+	var errs error
+	for _, id := range args {
+		if httpResp, err := c.V2Client.DeleteApiKey(id); err != nil {
+			errs = errors.Join(errs, errors.CatchApiKeyForbiddenAccessError(err, deleteOperation, id, httpResp))
+		} else {
+			output.Printf(errors.DeletedResourceMsg, resource.ApiKey, id)
```

Comment:
If the user deletes 10 resources, instead of having 10 separate lines, how about:
```
Deleted API keys "12345", "67890", ... and "13579".
```

Reply:
(it would use the same `plural()` logic as the "not found" error message)